scripts/init-ldes-data/app.mjs

```diff
@@ -17,24 +15,34 @@ process.env.TIME_TREE_RELATION_PATH =
 process.env.CACHE_SIZE = process.env.CACHE_SIZE || "10";
 process.env.DATA_FOLDER = process.env.DATA_FOLDER || "/project/data/ldes-feed";
 
+if (!process.env.BASE_URL?.length) {
+    console.error("You must provide a BASE_URL");
+    process.exit(1);
+}
 async function main() {
     const ldesProducerConfig = getConfigFromEnv();
     await deleteDirectory(process.env.DATA_FOLDER);
     const count = await getTotalCount();
-    const limit = 10000;
+    const limit = 100;
```

Comment:
why this change?

Reply:
we had bad triples in the triplestore in the old vlag dev environment that was making the script break, so I thought it would be better to make smaller batches and log the ones that fails to be investigated manually (instead of blocking 70k triples); that could be rollbacked though, I don't think the problem will occur in other environments

Reply:
> what do you mean with bad triple?

e.g:

```
<share://c7fb5ee5e347add29c1d36f31c1eeae92fc87ef440fba4ab95d7366652a2aa54.png>  <http://www.w3.org/1999/02/22-rdf-syntax-ns#type>       <http://xmlns.com/foaf/0.1/Image>
```
the subject is not allowed by the ldes producer library . This happens on vlag dev. this should not be of type "Image", but FileObject (the mu-file-service type)